overviews/implement-api-bindings.md

```diff
@@ -1,171 +0,0 @@
-# IPFS API Implementation Doc
```

Comment:
This looks like it may still be useful & accurate.

Reply:
@johnnymatthews we'd need to recycle this into the docs. Maybe along with the API reference?

Reply:
I opened https://github.com/ipfs/docs/issues/478

Reply:
It seems the docs are duplicated here https://github.com/ipfs/go-ipfs/blob/master/docs/implement-api-bindings.md